sys/dev/usb/usb_dev.h

```diff
@@ -123,6 +123,7 @@ struct usb_fifo {
 	void   *priv_sc0;		/* client data */
 	void   *priv_sc1;		/* client data */
 	void   *queue_data;
+	size_t fs_ep_sz;
```

Comment:
Could you use usb_size_t here? It makes it easier to compile the USB code for non-FreeBSD platforms and bootloaders?